processor/examples/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "module": "commonjs",
+
+    // The strict mode causes error for ts-node because it cannot find type definition for 'path-browserify' package
```

Comment:
We should not have any samples that aren't strict. Can we investigate the error more and hopefully fix it?

Reply:
I removed `path-browserify` based on the doc at https://github.com/browserify/path-browserify#install there's no deed to import from `path-browserify`.